streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -93,8 +92,7 @@
 
     private static final String WINDOWED_NAME = "KSTREAM-WINDOWED-";
 
-    private static final LongSerializer LONG_SERIALIZER = new LongSerializer();
-    private static final LongDeserializer LONG_DESERIALIZER = new LongDeserializer();
+    private static final SerDe<Long> LONG_SER_DE = SerDes.LONG();
```

Comment:
Nitpick: I think I'd call this `LONG_SERDE`.  Yes, the top-level class is called `SerDe` (capital D), but I think `SER_DE` is harder to read than `SERDE`.  Also, with `SER_DE` this won't show up when you'd want to look for "any serde stuff" in the code, e.g. via `grep -i` / `git grep -i` for "serde".